corehq/apps/reports/standard/monitoring.py

```diff
@@ -1792,22 +1801,74 @@ def _total_row(self, rows, report_data):
                     if report_data.submissions_by_user.get(user['user_id'], False):
                         active_users.add(user['user_id'])
                     all_users.add(user['user_id'])
-            total_row[3] = '%s / %s' % (len(active_users), len(all_users))
+            total_row[3] = {'numerator': len(active_users), 'demoninator': len(all_users)}
         else:
-            total_row[3] = '%s / %s' % (num, len(rows))
+            total_row[3] = {'numerator': num, 'denominator': len(rows)}
         return total_row
 
+    @property
+    def get_all_rows(self):
```

Comment:
Good point, I changed it here so that it only uses the old logic when `self.view_by_groups` is true: https://github.com/dimagi/commcare-hq/pull/23782/commits/a2290d9a82232dacc981120eda4f1e2a1b0eaf9d